homeworks/franktoth/5_oop/car_fleet_mgr.py

```diff
@@ -0,0 +1,71 @@
+#Creating Car class with the given attributes and methods
+
+class Car:
+    def __init__(self, brand, model, year):
+        self.brand = brand
+        self.model = model
+        self.year = year
+        self.mileage = 0
+        self.fuel_level = 100
+
+    def drive(self, distance):
+        fuel_consumption = distance * 0.1
+        if fuel_consumption <= self.fuel_level:
+            self.mileage += distance
+            self.fuel_level -= fuel_consumption
+            print(f"Car {self.brand} {self.model} drove {distance} km.")
+        else:
+            print(f"Car {self.brand} {self.model} doesn't have enough fuel to drive {distance} km.")
+
+    def refuel(self, amount):
```

Comment:
hasonló a helyzet, is vizsgál meg kérlek hogy nem e negatív az amount.

Reply:
Itt is módosítottam a scriptet, hogy ne lehessen negatív üzemanyagot tankolni.